### PR TITLE
I have added one element to ViewProfile.page.js and one test to viewP…

### PR DESCRIPTION
**Pull request by JuliaEde**
…rofile.spec.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "wdio": "npx wdio wdio.conf.js --spec",
         "test": "npm run clean && wdio && npm run report",
         "test:windows": "npm run clean:windows && wdio && npm run report",
-        "clean": "rm -rf allure-results && rm -rf allure-report",
+        "clean": "del -rf allure-results && rm -rf allure-report",
```
Comment (yuliahar) [inline package.json:10]: Please explain why do you change for mac OS clean script rm to del? Did del command work for windows? Please keep in mind that clean:windows script is for Windows.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (JuliaEde) [reply package.json:10]: Yes, rm throws error and I googled it, it says change rm to del to solve the problem.

Comment (JuliaEde) [reply package.json:10]: If you think it is useless please ignore the change, thank you!

Comment (yuliahar) [reply package.json:10]: Let's discuss it tomorrow.

Comment (JuliaEde) [reply package.json:10]: Okay, thank you!

Comment (JuliaEde) [reply package.json:10]: I don't really know what is it for, it just wasn't working with rm when I tried to run them

**Pull request merged**